packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -1213,8 +1213,9 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
                     ...options,
                 });
             },
-            pushData: async (...args: Parameters<Dataset['pushData']>) => {
-                return this.pushData(...args);
+            addRequests: this.addRequests,
```

Comment:
this surely deserves some tests, e.g. I am not sure if this will work without explicit binding (I'd guess it won't)

Reply:
Yeah, it won't. I'll just make it an arrow expression I guess.